src/ucp/rndv/rndv_put.c

```diff
@@ -405,6 +406,34 @@ ucp_proto_rndv_put_zcopy_query(const ucp_proto_query_params_t *params,
                       UCP_PROTO_ZCOPY_DESC, put_desc);
 }
 
+static void ucp_rndv_put_zcopy_proto_abort(ucp_request_t *request,
+                                           ucs_status_t status)
+{
+    ucp_request_t *rreq;
+
+    switch (request->send.proto_stage) {
+    case UCP_PROTO_RNDV_PUT_ZCOPY_STAGE_SEND:
+    case UCP_PROTO_RNDV_PUT_STAGE_FLUSH:
+        ucp_proto_request_zcopy_abort(request, status);
+        break;
+    case UCP_PROTO_RNDV_PUT_STAGE_ATP:
+    case UCP_PROTO_RNDV_PUT_STAGE_FENCED_ATP:
+        rreq = ucp_request_get_super(request);
+        ucs_assert(rreq->recv.length == rreq->recv.tag.info.length);
+        /* Locally the data is received, can complete with OK, but memory
+         * invalidation is not implemented in DC, so need to fail request to
```

Comment:
afaik, this is already  supported
@evgeny-leksikov

Reply:
better to double check by io-demo + interference  + DC + protov2 + put rndv scheme

Reply:
This still isn't supported. This PR enables support of this: #9206